src/api/todos.ts

```diff
@@ -0,0 +1,22 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const deleteData = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const createTodo = ({ title, userId, completed }: Omit<Todo, 'id'>) => {
+  return client.post<Todo>('/todos', { title, userId, completed });
+};
+
+export const updateT = ({ id, title }: Omit<Todo, 'userId' | 'completed'>) => {
+  return client.patch<Todo>(`/todos/${id}`, { title });
+};
+
+export const updateC = ({ id, completed }: Omit<Todo, 'userId' | 'title'>) => {
+  return client.patch<Todo>(`/todos/${id}`, { completed });
+};
```

Comment:
looks like it can be the same function:
```suggestion
type UpdateTodoInput = Partial<Omit<Todo, 'id' | 'userId'>>
export const updateTodo = (todo: UpdateTodoInput): Todo => {
  return client.patch<Todo>(`/todos/${id}`, todo);
};
```